### ISSUE TITLE
Fishnet 3

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#7]
Dataset: Fishnet
Data: 100%
Model: Best iNat21 with hierarchical multitask pretraining

Use a pre-trained swin-v2-base model pre-trained on iNat21 with hierarchical multitask loss with the best validation loss, fine-tune it on Fishnet, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).

**Comment by samuelstevens** [comment]
12 epochs later:

![image](https://user-images.githubusercontent.com/26638161/200446400-f89e1952-6d39-42c0-b93c-7ee34e8dbe21.png)

```

2022-11-07 23:26:49,546 - mmdet - INFO - Evaluating bbox...
Loading and preparing results...
DONE (t=0.86s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=43.22s).
Accumulating evaluation results...
DONE (t=9.55s).
2022-11-07 23:27:44,515 - mmdet - INFO -
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.350
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=1000 ] = 0.531
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=1000 ] = 0.411
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.013
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = 0.245
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = 0.387
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.461
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=300 ] = 0.461
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=1000 ] = 0.461
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.038
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = 0.354
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = 0.485

2022-11-07 23:27:45,429 - mmdet - INFO - Exp name: groovy_grape_fishnet.py
2022-11-07 23:27:45,430 - mmdet - INFO - Epoch(val) [12][2684]  bbox_mAP: 0.3500, bbox_mAP_50: 0.5310, bbox_mAP_75: 0.4110, bbox_mAP_s: 0.0130, bbox_mAP_m: 0.2450, bbox_mAP_l: 0.3870, bbox_mAP_copypaste: 0.350 0.531 0.411 0.013 0.245 0.387
```
